src/azure-cli/azure/cli/command_modules/appconfig/_params.py

```diff
@@ -326,3 +347,27 @@ def load_arguments(self, _):
 
     with self.argument_context('appconfig replica create') as c:
         c.argument('location', arg_type=get_location_type(self.cli_ctx), help='Location at which to create the replica.')
+
+    with self.argument_context('appconfig snapshot create') as c:
+        c.argument('snapshot_name', arg_type=snapshot_name_arg_type)
+        c.argument('filters', arg_type=snapshot_filter_arg_type)
+        c.argument('composition_type', options_list=['--composition-type'], arg_type=get_enum_type(["all", "group_by_key"]), help='Composition type used in building app configuration snapshots.')
```

Comment:
For enumerated values, we usually recommend using camel case instead of snake case
```suggestion
        c.argument('composition_type', options_list=['--composition-type'], arg_type=get_enum_type(["All", "GroupByKey"]), help='Composition type used in building app configuration snapshots.')
```

Reply:
@zhoxing-ms Snake case was used to maintain consistency with other enumerated value options like selecting [field parameters in output](https://github.com/Azure/azure-cli/blob/dev/src/azure-cli/azure/cli/command_modules/appconfig/_params.py#:~:text=arg_type%3Dget_enum_type(%5B%27key%27%2C%20%27value%27%2C%20%27label%27%2C%20%27content_type%27%2C%20%27etag%27%2C%20%27tags%27%2C%20%27locked%27%2C%20%27last_modified%27%5D)). Since users already specify other options, I believe this would be in line with the usage pattern.

Reply:
OK, make senses